website/pages/docs/integrate/nia-integration.mdx

```diff
@@ -87,15 +87,15 @@ Once the module development has been completed another email should be sent to n
 
 At this stage, it is expected that the module is fully developed, all tests and documentation are in place, and that HashiCorp has reviewed the module to be compatible with Consul-Terraform-Sync.
 
-Once this is done HashiCorp will get the new module listed as Consul-Terraform-Sync compatible on [consul.io](/use-cases/network-infrastructure-automation), and on [Terraform Registry](https://registry.terraform.io/browse/modules).
+Once this is done, HashiCorp will get the new module listed as Consul-Terraform-Sync compatible on [consul.io](/docs/nia/installation/requirements#using-terraform-modules), and then the Partner will be asked to publish the Terrform Module to the [Terraform Registry](https://registry.terraform.io/browse/modules).
```

Comment:
> then the Partner will be asked to publish the Terrform Module

then the partner will be asked to publish the Terraform module

Reply:
Fixed - thank you Kim!

Reply:
Ok, I think I finally got it :)